Week3/homework/step2-6.js

```diff
@@ -4,13 +4,17 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function printArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  for (let i = 0; i < arr.length; i++) {
+    console.log(arr[i].join(''));
+  }
```

Comment:
The code we expected you to write is:

```js
function printArray2d(arr) {
  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr[i].length; j++) {
      console.log(arr[i][j]);
    }
  }
}
```

However, your code passes the unit tests, so it would be unfair to reject it here.